crypto-ffi/bindings/android/build.gradle.kts

```diff
@@ -7,17 +7,32 @@ plugins {
 }
 
 val kotlinSources = projectDir.resolve("../jvm/src")
+val kotlinMain = projectDir.resolve("main/kotlin")
+
 val dokkaHtmlJar = tasks.register<Jar>("dokkaHtmlJar") {
     dependsOn(tasks.dokkaGeneratePublicationHtml)
     from(tasks.dokkaGeneratePublicationHtml.flatMap { it.outputDirectory })
     archiveClassifier.set("html-docs")
 }
 
+val generatedDir = layout.buildDirectory.dir("generated/uniffi").get().asFile
+val generatedMain = generatedDir.resolve("main/kotlin")
+val generatedTest = generatedDir.resolve("test/kotlin")
+
+val copyBindings by tasks.register<Copy>("copyBindings") {
```

Comment:
Why don't we output uniffi files directly into the `generatedMain` from the Makefile?

Reply:
We maybe could, but moving the uniffi-generated files in the Makefile seems likely to break the jvm target. 

Not sure if you saw the commit message attached to this one: https://github.com/wireapp/core-crypto/commit/fa0d053130570e7495f1c3ee18487b0d2f281a83; it feels relevant.